src/share/classes/com/sun/btrace/util/templates/Template.java

```diff
@@ -152,7 +155,9 @@ public String toString() {
     }
 
     static String getId(String owner, String name, String sig) {
-        return owner + "#" + name + "sig";
```

Comment:
Right. This one is not strictly necessary. Would be interesting to see what would be the result with the Java 9 indyfied string concatenation.